README.md

```diff
@@ -20,40 +20,55 @@ Currently an incomplete **pre-alpha**.
 
 ## Setup
 
-The Cosmic desktop environment requires a few dependencies:
+The COSMIC desktop environment requires a few dependencies:
 (This list does not try to be exhaustive, but rather tries to provide a decent starting point. For detailed instructions check out the individual projects):
 
 - [just](https://github.com/casey/just)
-- rust
-- libwayland
+- rustc
+- libwayland-dev
 - mesa (or third-party libEGL/libGL implementations, though interfacing with mesa's libglvnd is generally recommended).
-- libseat
-- libxkbcommon
-- libinput
+- libseat-dev
+- libxkbcommon-dev
+- libinput-dev
+- libgtk-4-1
 - udev
-- gtk4
 - dbus
 
 optionally (though the build-system might currently require these libraries):
-- libsystemd
-- libpulse
+- libsystemd-dev
+- libpulse-dev
 - pop-launcher
-- libexpat1-dev,
-- libfontconfig-dev,
-- libfreetype-dev,
-- lld,
+- libexpat1-dev
+- libfontconfig-dev
+- libfreetype-dev
+- lld
+
```

Comment:
This section seems Pop!_OS/Ubuntu (and maybe Debian?) specific.

Could you add a sub-header for this to note, that this is distribution specific.